src/tests/compiler/tools/languageServiceTests.ts

```diff
@@ -112,6 +112,57 @@ describe(nameof(LanguageService), () => {
             });
         });
     });
+
+    describe(nameof<LanguageService>(l => l.getEditsForRefactor), () => {
+        it("should get edits for known refactor 'move to a new file'", () => {
+            const { sourceFile, project } = getInfoFromText("export class A {}; function f(){return new A(); }", { filePath: "/file.ts" });
+            const classId = sourceFile.getClassOrThrow("A").getNameNodeOrThrow();
+            const results = project.getLanguageService().getEditsForRefactor(sourceFile.getFilePath(), {},
+                { pos: classId.getStart(), end: classId.getEnd() }, "Move to a new file", "Move to a new file", {});
+
+            const edit1 = results!.getEdits().find(edit => edit.getFilePath() === sourceFile.getFilePath());
+            const edit2 = results!.getEdits().find(edit => edit.getFilePath() === "/A.ts");
+
+            checkFileTextChanges(edit1!, {
+                fileName: "/file.ts",
+                textChanges: [{
+                    newText: "import { A } from \"./A\";\n\n",
+                    span: { start: 0, length: 0 }
+                }, {
+                    newText: "",
+                    span: { start: 0, length: 17 }
+                }]
+            });
+
+            checkFileTextChanges(edit2!, {
+                fileName: "/A.ts",
+                textChanges: [{
+                    newText: "export class A {\n}",
+                    span: { start: 0, length: 0 }
+                }]
+            });
+        });
+    });
+
+    describe(nameof<LanguageService>(l => l.getCodeFixesAtPosition), () => {
+        it("should get code fixes at position for known code fixes convertToEs6Module (error code 80001)", () => {
+            const { sourceFile, project } = getInfoFromText("const moment = require('moment'); moment(); ", { filePath: "/file.ts" });
+            const variableDeclaration = sourceFile.getVariableDeclarationOrThrow("moment");
+            const results = project.getLanguageService().getCodeFixesAtPosition(sourceFile.getFilePath(), variableDeclaration.getStart(), variableDeclaration.getEnd(), [80001], {}, {});
+
+            expect(results[0]!.getFixName()).to.equal("convertToEs6Module");
+            expect(results[0]!.getDescription()).to.equal("Convert to ES6 module");
+
+            checkFileTextChanges(results[0]!.getChanges()[0], {
```

Comment:
Does this only return 1 change? Could you add an assertion for how many results this should return?